tests/test_rank_genes_groups.py

```diff
@@ -307,6 +307,13 @@ def test_wilcoxon_tie_correction(reference):
     np.testing.assert_allclose(test_obj.stats[groups[0]]["pvals"], pvals)
 
 
+def test_wilcoxon_huge_data(monkeypatch):
```

Comment:
Yes! I love monkey patching!

Reply:
Haha in some cases it’s really nice, such as here where I don‘t really want to create an AnnData object with >10M cells in our test suite!